.github/CODEOWNERS

```diff
@@ -122,7 +122,7 @@
 
 # PRLabel: %Cognitive - Language
 # ServiceLabel: %Cognitive - Language %Service Attention
-/sdk/cognitivelanguage/                         @heaths @rokulka
+/sdk/cognitivelanguage/                         @heaths @rokulka @mshaban-msft
```

Comment:
Do partner contacts for "Service Attention" need to be updated for this and the Translator change?

Reply:
No. In case of cognitive, service team is different from the service SDK team.